pkg/kapp/config/default.go

```diff
@@ -700,10 +703,24 @@ changeRuleBindings:
       - hasNamespaceMatcher: {}
 `
 
-var defaultConfigRes = ctlres.MustNewResourceFromBytes([]byte(defaultConfigYAML))
-
 func NewDefaultConfigString() string { return defaultConfigYAML }
 
 func NewConfFromResourcesWithDefaults(resources []ctlres.Resource) ([]ctlres.Resource, Conf, error) {
-	return NewConfFromResources(append([]ctlres.Resource{defaultConfigRes}, resources...))
```

Comment:
Does this help in significant performance improvement? (I hope we are not reducing code readability for an insignificant amount of improvement)